.circleci/config.yml

```diff
@@ -52,12 +55,9 @@ jobs:
     executor: node_executor
     steps:
       - checkout
-      - run:
-          name: update-npm
-          command: 'sudo npm install -g npm@latest'
       - restore_cache:
           keys:
-            - deps-{{ checksum "package.json" }}
+            - deps-{{ checksum "./package.json" }}
```

Comment:
Either have the `./` as here or not as per the `requirements.txt` above.
Otherwise looks good

Reply:
done